### DISCUSSION TITLE
It is possible that my proxy is missing some kind of traffic?

### DISCUSSION THREAD

**Discussion by henriquesati** [mitmproxy/mitmproxy#7253] [General]
Hello, i'm trying to intercept connections from a application running on windows using its windows proxy config (to 127.0.0.1:8080) and mitmproxy, to see how it is identifying that i'm using multiple accounts on the same device. I'm being able to apparently see the requests on mitmproxy, but i'm not identifying what I was expecting, some kind of device id identifier or random id being sent on requests. so there is any kind of connections which can be done outside the proxy and I can't see it?

**Comment by mhils** [comment]
Windows applications can generally bypass the proxy settings configured in Windows. I would recommend you try out local redirect mode instead. :)

**Reply by Srijan67** [reply]
the issue with local redirect mode is it require UAC permission whenever we trigger it in that mode. Isn't it. Can we suppress it or do something about it.

**Reply by mhils** [reply]
You can start mitmproxy itself as root, it shouldn't ask for permissions again then.
